apps/docs/src/components/home/built-in-stitches.tsx

```diff
@@ -3,17 +3,15 @@ import NextLink from 'next/link';
 import { InView } from 'react-intersection-observer';
 import cn from 'classnames';
 import { Section, Title, Subtitle, BlockLink, Box } from '@primitives';
-import {
-  Grid,
-  Row,
-  Col,
-  Link,
-  Text,
-  Collapse,
-  Spacer
-} from '@nextui-org/react';
+import * as react from '@nextui-org/react';
 import landingContent from '@content/landing';
-import { CodeDemo, Blockholder } from '@components';
+import { Blockholder } from '@components';
+
+import dynamic from 'next/dynamic';
+
+const DynaminCodeDemo = dynamic(() => import('../code-demo/code-demo'), {
```

Comment:
Hey @TIMMLOPK thanks for contributing to NextUI

Is not necessary to dynamically import the `code-demo` component due to we are already showing it just when is visible in the viewport line:49